pkgs/development/python-modules/curl-cffi/default.nix

```
@@ -1,29 +1,31 @@
 {
-  stdenv,
   lib,
-  buildPythonPackage,
   fetchFromGitHub,
   setuptools,
   addBinToPathHook,
+  python,
   curl-impersonate-chrome,
-  cffi,
-  certifi,
-  charset-normalizer,
-  cryptography,
-  fastapi,
-  httpx,
-  proxy-py,
-  pytest-asyncio,
-  pytest-trio,
-  pytestCheckHook,
-  python-multipart,
-  trustme,
-  uvicorn,
-  websockets,
   writableTmpDirAsHomeHook,
 }:
 
-buildPythonPackage rec {
+let
+  customPython = python.override {
+    packageOverrides = self: super: {
+      websockets = super.websockets.overridePythonAttrs (old: rec {
```

```suggestion
  websockets = super.websockets.overridePythonAttrs (old: rec {
```

Doing this in python*Packages is not allowed. Only applications outside of pythonPackages can do that.

---

Well, nuts. I'll pin it in a `let`. Thank you.